lib/Job.js

```diff
@@ -41,6 +41,21 @@ function Job(name, job, callback) {
     this.callback = typeof callback === 'function' ? callback : false;
   }
 
+  // task count
+  this.running = 0;
+
+  this.on('run', function () {
```

Comment:
I'm slightly concerned about generating three listeners per each job and then never removing them. Isn't that going to cause memory leaks, especially when jobs are being created dynamically?
Is there no place in the code where we could manage these counters without using listeners?